src/brogue/RogueMain.c

```diff
@@ -1233,7 +1245,17 @@ void victory(boolean superVictory) {
     sprintf(buf, "%li", totalValue);
     printString(buf, mapToWindowX(60), min(ROWS-1, i + 1), &lightBlue, &black, dbuf);
 
-    i += 4;
+    funkyFade(dbuf, &white, 0, 120, COLS/2, ROWS/2, true);
+    displayMoreSign();
+    clearDisplayBuffer(dbuf);
+    deleteMessages();
+
+    //
+    // Third screen - List of achievements
+    //
+    printString("Achievements", mapToWindowX(0), mapToWindowY(-1), &white, &black, dbuf);
```

Comment:
It would be good if you can align this better with previous screen text, with the same margins (probably using -1 isn't a great idea), and maybe using `lightBlue` colour, etc.

Reply:
I wrote "achievements" but I thought you would like to write a other sentence. I will work on presentation.

Reply:
"Achievements" is good! I just mean its placement and colour really. It's more of a style thing, so you can leave it as it is, and I will add a commit before merging that tweaks things to my liking :P

Reply:
Ok :p